app/models.py

```diff
@@ -0,0 +1,42 @@
+from app import db
+from werkzeug.security import generate_password_hash, check_password_hash
+
+
+class User(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    username = db.Column(db.String(64), index=True, unique=True)
+    email = db.Column(db.String(64), index=True, unique=True)
+    email_verified = db.Column(db.Boolean)
+    password_hash = db.Column(db.String(1024))
+
+    def set_password(self, password) -> None:
+        self.password_hash = generate_password_hash(password)
+
+    def check_password(self, password) -> bool:
+        return check_password_hash(self.password_hash, password)
+
+class House(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    invite_link = db.Column(db.String(64), index=True, unique=True)
+    name = db.Column(db.String(64), index=True, unique=True)
+    description= db.Column(db.String(1024))
+
+    def update_invite_link(self, link) -> None:
```

Comment:
I don't think this is so useful, because `<house instance>.invite_link = link` is just as good?